doc/articles/features/pointers-keyboard-and-other-user-inputs.md

```diff
@@ -0,0 +1,183 @@
+| `DropCompleted`               | Yes     | Yes     | Yes     | Yes   | Yes      | Yes      | Yes   | [Documentation](https://docs.microsoft.com/uwp/api/windows.ui.xaml.uielement.dropcompleted) |
+
+
+Notes:
+
+1. **Focus** events:
+   * **iOS**: The concept of _focus_ is emulated because not supported by the platform, so this event is
+     always bubbling in managed code.
+   * **Wasm**: Current implementation is not totally reliable and doesn't support _lost focus_ most of the time.
+2. **Keyboard** events:
+   * **Android**: `KeyDown` and `KeyUp` events are **generated only from hardware keyboards** (Except for the _Editor Action_ on _soft keyboards_,
+     those are translated as `KeyUp` with `KeyCode.Enter`). Some soft keyboard **MAY** generate those events, but your code shouldn't rely
+     on that. This is a limitation [in the Android platform](https://developer.android.com/training/keyboard-input/commands) (see note on this link content).
+     > Because of those limitations, _Key Events_ are not being implemented as _routed events_ on Android, so `AddHandler` & `RemoveHandler`
+     > won't work for keyboard events. **They won't bubble in managed code**.
+   * **iOS**: `KeyDown` & `KeyUp` routed events are generated from only a `TextBox`. Only character-related keyboard events are generated.
+     They are implemented as _Routed Events_ and they are **always bubbling in managed code**.
+
+## Pointer Events
+
+Those events are the base for all others pointing device related events (i.e. Manipulation, Gesture and drag and dop events).
+They are directly linked to the native events of each platform:
+* `Touches[Began|Moved|Ended|Cancelled]` on iOS
+* `dispatchTouchEvent` and `dispatchGenericMotionEvent` on Android
+* `pointer[enter|leave|down|up|move|cancel]` on WebAssembly
+
+On skia however, they are fully managed events.
+
+### Pointers events and the ScrollViewer
+
+Like on WinUI as soon as the system detects that the user wants to scroll, a control gets a `PointerCancelled` and that control won't receive
+any other pointer event until the user release the pointer. That behavior can be prevented by setting the `ManipulationMode` 
+to something else than `System` on a control nested in the `ScrollViewer`. (cf. [Manipulation events](#Manipulation_Events))
+
+Be aware that on iOS this will set `DelaysContentTouches` to `false` so it means that it will slightly reduce the performance
+of the scrolling (cf. [documentation](https://developer.apple.com/documentation/uikit/uiscrollview/1619398-delayscontenttouches)).
+
+### Known limitations for pointer events
+
+As those events are tightly coupled to the native events, Uno has to make some compromises:
+* On iOS, when tapping with a mouse or a pen on Android, or in few other specific cases (like `PointerCaptureLost`), 
+  multiple managed events are raised from a single native event. These have multiple effects:
+	* On UWP if you have a control A and a nested control B, you will get:
+		```
```

Comment:
Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, add 1 space](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6334034)